docs/geedocs/answer/7160000.html

```diff
@@ -44,7 +44,7 @@
   <p>The Open GEE 5.2.0 release is supported on 64-bit versions of the following operating systems:</p>
     <ul>
       <li>Red Hat Enterprise Linux version 7.x, including the most recent security patches</li>
-      <li>CentOS 7.x</li>
```

Comment:
I'd say back out all the changes to this file.  This is the release notes for 5.2.0, and 5.2.0 doesn't support CentOS 6 or RHEL 6.